City-of-Pori/pori-kada
# PORI-373 Moved visitpori theme liftup context under correct feature, …

…removed image from incorrect place

To test:

1. drush fra -y && drush cc all
2. Go to for example https://local.visitpori.fi/nae-ja-koe/ostokset and make sure the liftup image is not displayed on the page.
